docs/UserGuide.md

```diff
@@ -137,11 +149,11 @@ designers, and boutique flower businesses.
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `addressbook.jar` from [here](https://github.com/AY2324S2-CS2103T-T09-2/tp/releases).
+1. Download the latest `bookkeeper.jar` from [here](https://github.com/AY2324S2-CS2103T-T09-2/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your AddressBook.
+1. Copy the file to the folder you want to use as the _home folder_ for your BookKeeper.
 
-1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar addressbook.jar`
+1. Open a command terminal, `cd` into the folder you put the jar file in, and use the `java -jar bookkeeper.jar`
```

Comment:
Need to double check it's always `BookKeeper.jar` (case sensitive) rather than `bookkeeper.jar`
Also double check whether we can change the filename in v1.4 uh

Reply:
We will keep it as lowercase "bookkeeper.jar"